BiasAndFairnessModule/run_tests.py

```
@@ -0,0 +1,173 @@
+    print("1. Testing basic imports...")
+    try:
+        import numpy as np
+        print("   ✅ NumPy imported successfully")
+    except ImportError as e:
+        print(f"   ❌ NumPy import failed: {e}")
+        return False
+    
+    # Test 2: Metric registry
+    print("2. Testing metric registry...")
+    try:
+        from src.eval_engine.metric_registry import list_metrics, get_metric
```

Ruff (0.12.2)

`src.eval_engine.metric_registry.get_metric` imported but unused

Remove unused import: `src.eval_engine.metric_registry.get_metric`

(F401)